### PR TITLE
Add module to wrap PrairieLib and initialize the database

### PR DESCRIPTION
**Pull request by Binary-Eater**
…b driver

Also, should init a postgres database that has been set up with `dbInit.sql`

### CODE REVIEW

**Review by JParisFerrer — APPROVED** [review]
there is code

File: src/db-driver.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+const pgConf = {
+  host: config.postgresqlHost,
+  port: config.postgresqlPort,
+  user: config.postgresqlUser,
+  password: config.postgresqlPassword,
+  database: config.postgresqlDatabase,
+}
+
+dbDriver.initDB = function(callback, idleErrCb) {
```
Comment (nwalters512) [inline src/db-driver.js:18]: It looks like `callback` is never invoked if there isn't an error?

File: src/server.js [hunk 2]
```diff
@@ -21,5 +21,14 @@ app.use(
 )
 
 const server = Server(app)
-server.listen(PORT)
-logger.info(`Listening on ${PORT}`)
+server.listen(config.PORT)
```
Comment (nwalters512) [inline src/server.js]: We can't begin accepting connections until the database is set up.

File: src/db-driver.js (hunk 1 above)
Comment (Binary-Eater) [reply src/db-driver.js:18]: Sorry, on my phone, line 34.

File: src/server.js (hunk 2 above)
Comment (Binary-Eater) [reply src/server.js]: Ya, my bad. Didn't think about it since it was on the server-side, not db-side. Will fix when I get a chance to sit down.

**Pull request merged**